### PR TITLE
added error packet for when unable to write to SD

### PR DESCRIPTION
**Pull request by vainaviv**

### CODE REVIEW

**Comment by vainaviv** [comment]
I updated the E1 Design spreadsheet and included the codes for error packets and assigned SD write error to 101. All errors will be in the 100s

File: Brain/Brain.ino [hunk 1]
```diff
@@ -58,6 +58,7 @@ const char * file_name = str_file_name.c_str();
 
 long startTime;
 String packet;
+String error_packet;
```
Comment (fazerlicourice7) [inline Brain/Brain.ino]: don't think we need a separate variable for this; just use packet. It's a transient variable anyway, it changes on every iteration

**Review by fazerlicourice7 — COMMENTED** [review]
just one tiny change. And hold off on merging for a sec. Let's see what the resolution of discussion in slack is.

File: Brain/Brain.ino [hunk 2]
```diff
@@ -128,12 +132,9 @@ void loop() {
     board_address = sensor.board_address;
     sensor_id = sensor.id;
     sensorReadFunc(sensor.id);
-    packet = make_packet(sensor.id);
+    packet = make_packet(sensor.id, false);
     Serial.println(packet);
     RFSerial.println(packet);
-    if (!write_to_SD(packet.c_str())){
```
Comment (fazerlicourice7) [inline Brain/Brain.ino:134]: Still need to actually writeToSD, just don't do anything w/ the results

**Comment by fazerlicourice7** [comment]
and fix merge conflicts

File: Brain/Brain.ino (hunk 2 above)
Comment (vainaviv) [reply Brain/Brain.ino:134]: Oops added that back in

**Pull request merged**